restcomm/restcomm.mscontrol.jsr309/src/main/java/org/restcomm/connect/mscontrol/jsr309/Jsr309CallController.java

```diff
@@ -810,15 +809,20 @@ private void onStop(Stop message, ActorRef self, ActorRef sender) {
                         builder.setAccountSid(accountId);
                         builder.setCallSid(callId);
                         builder.setDuration(duration);
-                        builder.setApiVersion(runtimeSettings.getString("api-version"));
-                        StringBuilder buffer = new StringBuilder();
-                        buffer.append("/").append(runtimeSettings.getString("api-version")).append("/Accounts/")
-                                .append(accountId.toString());
-                        buffer.append("/Recordings/").append(recordingSid.toString());
-                        builder.setUri(URI.create(buffer.toString()));
+                        String apiVersion = runtimeSettings.getString("api-version");
```

Comment:
shall we replace hardcoded "api-version" with String constant that can be reused?